src/pip/_internal/cli/req_command.py

```diff
@@ -50,14 +50,12 @@ class SessionCommandMixin(CommandContextMixIn):
     A class mixin for command classes needing _build_session().
     """
 
-    def __init__(self):
-        # type: () -> None
+    def __init__(self) -> None:
         super().__init__()
         self._session = None  # Optional[PipSession]
```

Comment:
This can be converted to `self._session: Optional[PipSession] = None`.

Reply:
Understood. I will aply it.